internal/processmon/processmon.go

```diff
@@ -287,17 +288,18 @@ func (p *ProcessMon) LaunchProcess(contextID string, refPid int, refNSPath strin
 			zap.Error(pidstaterr),
 		)
 	}
-	_, staterr := os.Stat(netnspath)
+	_, staterr := os.Stat(p.netnspath)
 	if staterr != nil {
-		mkerr := os.MkdirAll(netnspath, os.ModeDir)
+		mkerr := os.MkdirAll(p.netnspath, os.ModeDir)
 		if mkerr != nil {
 			zap.L().Error("Could not create directory", zap.Error(mkerr))
 		}
 	}
 
 	// A symlink is created from /var/run/netns/<context> to the NetNSPath
-	if _, lerr := os.Stat(filepath.Join(netnspath, contextID)); lerr != nil {
-		linkErr := os.Symlink(nsPath, netnspath+contextID)
+	contextFile := filepath.Join(p.netnspath, contextID)
+	if _, lerr := os.Stat(contextFile); lerr != nil {
```

Comment:
if you don't need contextFile after that 

```go
if _, err = os.Stat(filepath.Join(p.netnspath, contextID)); err != nil
```

Reply:
done